Travel Scheduler/Test/APITesting.m

```diff
@@ -34,7 +34,7 @@ +(void)testGetId {
             NSLog(@"did not work snif");
```

Comment:
Don't know if this has been resolved already. But NSLog could be removed if code is finalized.